src/osd/PG.cc

```diff
@@ -3847,6 +3850,30 @@ void PG::replica_scrub(
     return;
   }
 
+  scrubber.active_rep_scrub = op;
```

Comment:
I think this is wrong, looks like active_rep_scrub should only be set if we need op_applied or _applied_recovered_object_replica to requeue the op.